Avi

avihpsharma opened the pull request:
This is basic initial version of java application to perform various operation on linked list.

Command to run application with jar:
java -jar jarName

avihpsharma committed:
Merge pull request #1 from HammadMasood/master

Adding RotateList Project

avihpsharma committed:
Initial verision of application to perform various functions on linked list

avihpsharma committed:
Merge branch 'avi' of https://github.com/HammadMasood/GitPlayground into avi

avihpsharma committed:
Added delete Node and append option in application

avihpsharma committed:
adding new change

avihpsharma committed:
Adging more

HammadMasood reviewed (changes requested):
Don't commit class files, try to exclude them by adding them in .gitignore

avihpsharma committed:
test

avihpsharma committed:
test .gitignore file

avihpsharma commented:
I have tried .gitignore concept, its working fine.
Please review again and merge the changes.

HammadMasood committed:
Merge pull request #4 from HammadMasood/avi

Avi

HammadMasood committed:
Following changes done:
1. Removed null from the end
2. Added constructor to take string array argument
3. Added two TODO items

HammadMasood committed:
Added .classpath in gitignore

avihpsharma committed:
Merge pull request #5 from HammadMasood/hammad

Removed Null from the end

avihpsharma commented:
Changes look excellent.
